Загальний коміт

LisWar opened the pull request:
- Іконки стрілок у випадаючому меню моб. версії не працюють. Прошу виправити.
- Не зовсім зрозуміло, як регулювати відступи між елементами хедеру на табл/десктопній версіях.
- Змінив глобальну перемінну mobile, щоб відображати максимальну ширину мобільної версії.

LisWar committed:
Загальний коміт

- Іконки стрілок у випадаючому меню моб. версії не працюють. Прошу виправити.
- Не зовсім зрозуміло, як регулювати відступи між елементами хедеру на табл/десктопній версіях.
- Змінив глобальну перемінну mobile, щоб відображати максимальну ширину мобільної версії.

LisWar committed:
Для повторного коміта

Не додав рев'юерів

ValentynPakhniuk reviewed (commented):
Привіт! Неможу зарахувати твою роботу туму що потрібно пофіксить конфлікти і невірно виконана!

LisWar committed:
feature/styles

- Додав заново стилі окрім:
1) іконок у випадаючому меню - треба HTML.
2) стрілочок у кінці пунктів цього ж меню, та сама причина.
3) поточної сторінки - треба JS
4) приховування іконки пошуку при відкритті моб. меню - можна чаклувати із CSS, в JS робиться за пів секунди.

- Повторно виправив @mobile на <767рх
- Додав наслідування кольору тексту для посилань (було відсутнє)

SCSS файл вийшов дуже великим, пропоную розбити на підмодулі, стягнути в один header.scss і вже його додавати в індекс.

Якщо дасте відмашку доробити із змінами у HTML/JS - дороблю.

LisWar committed:
Merge branch 'main' into header

LisWar commented:
Після мержу все посипалось до чортів. Шукаю шлях це виправити.
Нажаль, сьогодні вже не встигну, я так дивлюсь.
